### PR TITLE
sample docker-compose labkey20.3 community editon service

### PR DESCRIPTION
**Pull request by labkey-stuartm**
Initial commit of public docker-compose sample configuration

### CODE REVIEW

File: docker/labkey-20.3-community/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    1.  DB username and password should match those configued in labkey/config/labkey.xml 
+
+
+### Running LabKey Server 
+
+To build and run the LabKey service  
+
+    sudo docker-compose up -d --build
+
+After few seconds, open [http://localhost:8080/labkey](http://<host>:8080) to see the LabKey Server initialization page.
```
Comment (curiositycasualty) [inline docker/labkey-20.3-community/README.md]: I think you want `[http://localhost:8080/labkey](http://localhost:8080/labkey)` which will be a link to "http://localhost:8080/labkey".

File: docker/labkey-20.3-community/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+samples/docker/labkey-20.3-community
+==========
+
+This folder contains a sample docker-compose file and sample Docker configurations to stand up an instance of LabKey Server Community edition for testing.
+
+### IMPORTANT: Not Recommended for Production use
+ This image was meant for trying out LabKey Server and not meant for running a production server or for storing important biomedical data.
```
Comment (curiositycasualty) [inline docker/labkey-20.3-community/README.md]: "This image is meant for trying out LabKey Server and not meant for running LabKey as a production service."

File: docker/labkey-20.3-community/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+### Recommended configuration steps
+
+1. Clone this repo
+1. Edit the labkey/config/labkey.xml and choose your own settings for the following
+    1. jdbc (DB) username and password
+    1. MasterEncryptionKey password 
+    1. SMTP settings
+1. Edit the postgres/fixtures/create_fixtures.sql file
+    1.  DB username and password should match those configued in labkey/config/labkey.xml
```
Comment (curiositycasualty) [inline docker/labkey-20.3-community/README.md:25]: Maybe something here about the `docker-compose.yml` file also needing to match.

**Review by curiositycasualty — APPROVED** [review]
Just some minor suggestions, otherwise, assume it works, looks good.

File: docker/labkey-20.3-community/README.md (hunk 3 above)
Comment (labkey-stuartm) [reply docker/labkey-20.3-community/README.md:25]: the postgres password in docker-compose is for the postgres super user and not the db user.  which I also find odd...

File: docker/labkey-20.3-community/README.md (hunk 1 above)
Comment (labkey-stuartm) [reply docker/labkey-20.3-community/README.md]: Fixed

File: docker/labkey-20.3-community/README.md (hunk 2 above)
Comment (labkey-stuartm) [reply docker/labkey-20.3-community/README.md]: fixed

File: docker/labkey-20.3-community/README.md (hunk 3 above)
Comment (labkey-stuartm) [reply docker/labkey-20.3-community/README.md:25]: Added wording to address this and update the docker-compose.yml

**Pull request merged**